### ISSUE TITLE
sycli list: Allow to receive input

### ISSUE THREAD

**Issue by alexherbo2** [Luminarys/synapse#208]
It may be nice if `sycli list` could build an ASCII table from a JSON list.

```
sycli list | <filter> | sycli list
```

**Example** – Sorting:

``` sh
sycli list |
jq-sort-by-name |
sycli list
```

**Example** – Grepping:

``` sh
sycli list |
jq-select-one-piece-resources |
sycli list
```

**Comment by evanrichter** [comment]
cool idea, but I don't think sycli is meant to be a JSON -> ascii table tool

you could try using just `jq` itself like [this answer](https://stackoverflow.com/a/39139478)

**Comment by alexherbo2** [comment]
Yep, I think also it is out of the scope of `sycli`.  Just wanted to throw the idea because I actually wanted to do some filtering and it is not as easy as filtering lines with `sort` or `grep` (if we want to keep the ASCII table).

**Comment by evanrichter** [comment]
true, ascii tables aren't very unix like anyway. to fit in better with standard unix tools like `sort`/`grep` then we should just output tabulated data like the output of `ps`

**Comment by evanrichter** [comment]
the change here would be pretty minimal: prettytable-rs has an [undecorated format](https://docs.rs/prettytable-rs/0.8.0/prettytable/format/consts/struct.FORMAT_CLEAN.html) predefined

**Comment by Luminarys** [comment]
Yea I'm in favor here of allowing a raw output, as noted it should be a fairly simple change.
